qa/rpc-tests/rest.py

```diff
@@ -53,7 +53,8 @@ def __init__(self):
         self.num_nodes = 3
 
     def setup_network(self, split=False):
-        self.nodes = start_nodes(self.num_nodes, self.options.tmpdir)
+        args = ["-txoutsbyaddressindex"]
```

Comment:
Nit: You can put something like the following up a few lines, so the scope is not limited to `setup_network`.

```py
self.extra_args = [["-txoutsbyaddressindex"]] * 3